components/user/UserTabs.js

```diff
@@ -92,7 +92,7 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
           id="tabs"
           name="tabs"
           onChange={changeTab}
-          className="block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
+          className="block w-full rounded-md -mb-px border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
```

Comment:
What does this do `-mb-px`?

Reply:
This `-mb-px` is a part of original code and it adds margin to the bottom i.e. it refers to this 
https://github.com/EddieHubCommunity/LinkFree/pull/4531#discussion_r1098263721